Apps/Sandcastle/gallery/Clamp to 3D Tiles Offscreen.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
Are we sure we should call this "off screen?"  What speaks to the user best?  Async?  Most Detailed?

Reply:
Really just that comment for now, this is cool, and could be used for profiles, e.g., change `30` to `300`.

I'll also look at the implementation when ready.

Reply:
While I do support having smaller, more focused Sandcastle examples in general, in this case I wonder if it would be better to combine this with the existing Clamp to 3D Tiles example, in the same way the terrain sample and sampleMostDetailed are in the same example:

http://localhost:8080/Apps/Sandcastle/index.html?src=Terrain.html

"Most detailed" is probably the most relevant to the user, but I can imagine users running into this issue when they realize sampleHeight only works for already visible tiles and want to figure out how to do it for tiles offscreen. I think in any case the goal for the user is just clamping to 3D Tiles/getting tiles height, so a combined example geared towards that use case makes the most sense to me.

Reply:
I think keeping them separate is better because they represent different enough use cases.

But maybe the new demo should be called `Sample Height from 3D Tiles`. The main difference being this demo samples some heights and shows the result while the first demo actually physically clamps something to the tileset.

Reply:
I think that makes more sense! I think that naming will make it easier to find.